### ISSUE TITLE
No search suggestions

### ISSUE THREAD

**Issue by talha5758** [nsh07/WikiReader#10]
![Screenshot_2024-10-22-18-27-17-23_b315a416a476f0a63f3f279419fb5c93.jpg](https://github.com/user-attachments/assets/ca762458-9beb-4450-be4e-9315cb96127c)

**Comment by nsh07** [comment]
Yeah I haven't implemented search suggestions yet. Might work on it, but i kind of contradicts the goal of this app, simplicity. Having search suggestions would slow down the app quite a bit.

But let's see, I will create a new issue, and might implement it later

**Comment by nsh07** [comment]
Hey! Sorry for replying on such an old issue but I've implemented search suggestions and it would be available in the next alpha release and in the stable release of v2!
